utils/rpms/daos.spec

```diff
@@ -128,7 +128,7 @@ Requires: spdk-tools
 Requires: ndctl
 # needed to set PMem configuration goals in BIOS through control-plane
 %if (0%{?suse_version} >= 1500)
-Requires: ipmctl < 02.00.00.3809
+Requires: ipmctl >= 02.00.00.3733
```

Comment:
Why is this one lower than the EL7 minimum?  Which can be read in the inverse, why is the EL7 minimum higher than the Leap 15 minimum?  If the issue is in the core code and not packaging, the requirement could/should be the same on both, no?  Eliminating these kinds of unnecessary (if it is unnecessary) per-distro differences will help with the future maintainability of this file.

Reply:
leap15 currently ships with 3733, el7 ships with a more recent version, 3733 doesn't support the most recent (which may be pre-release) PMem firmware, so the thinking is we don't want to block leap15 users from installing with stock but we also don't want EL7 users from using something less than what is available in the distro (i.e. we want to force an upgrade if there is something more recent available)

Reply:
I tried to summarise in https://jira.hpdd.intel.com/browse/DAOS-6038 description

Reply:
> leap15 currently ships with 3733, el7 ships with a more recent version, 3733 doesn't support the most recent (which may be pre-release) PMem firmware, so the thinking is we don't want to block leap15 users from installing with stock but we also don't want EL7 users from using something less than what is available in the distro (i.e. we want to force an upgrade if there is something more recent available)

So just to boil this down into a plain english assertion (to confirm that it's really what we intend to assert), we are saying that if EL7 users cannot get ipmctl version > 3816 (but may have access to a less version < 3816 but > 3733) then we will not allow them to install daos.  But for Leap users, we will allow them to install daos as long as their ipmctl is > 3733 even though it does not have the most recent firmware.

I'm not arguing or advocating one way or the other.  I'm just confirming our intention here and making sure it is what we are asserting in our packaging.

Reply:
that is the correct assertion, EL7 users currently have access to >3816 in distro, leap15 users dont, leap15.3 is out but doesn't currently have an official package version for ipmctl yet: https://software.opensuse.org/package/ipmctl shows 3733 for 15.2 and 3847 for Tumbleweed . runtime checks in the control plane will prevent operations if one of the bad ipmctl versions is detected